targets/mbedos5/tools/js2c.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright (c) 2016 ARM Limited. All rights reserved.
```

Comment:
I'm not sure that the "all rights reserved" clause in the copyright line is compatible with the rest of the license.